hack/build.sh.d/kn-plugin-func.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+# There no midstream image currently, fallback to upstream one.
+readonly socat_image="${KN_PLUGIN_FUNC_SOCAT_IMAGE:-quay.io/boson/alpine-socat:1.7.4.3-r1-non-root}"
```

Comment:
I think this PR should also touch the RPM spec file.

Reply:
Indeed, thanks!